.github/workflows/reusable-windows-ci.yml

```
@@ -23,6 +23,11 @@ on:
         description: 'Branch or tag of Fast DDS repository (https://github.com/eProsima/Fast-DDS)'
         required: true
         type: string
+      fastcdr-branch:
```

In all the occurences
```suggestion
      fastcdr_branch:
```